manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -177,8 +175,6 @@ spec:
           securityContext:
             allowPrivilegeEscalation: false
             readOnlyRootFilesystem: true
-            runAsUser: 1337
-            runAsGroup: 1337
```

Comment:
Why is this removed? This will not be added back by injection, seems like a regression for non-openshift users

Reply:
It looks like these files don't get parsed by the template, so we can't use `{{ .ProxyUID }}` here, we can only rely on `.Values`. Right?

Reply:
Yes, they are not injected. They have nothing to do with ProxyUID technically either since they are not proxies... although maybe we want similar logic. 

AFAIK this is fixing openshift but in the process, making stock Kubernetes run as root which is not what we want.

Reply:
> AFAIK this is fixing openshift but in the process, making stock Kubernetes run as root which is not what we want.

No because the pilot image runs as user 1337.

Reply:
I mean, we are safe to remove these lines because https://github.com/istio/istio/blob/master/pilot/docker/Dockerfile.pilot#L25 and https://github.com/istio/istio/blob/master/operator/docker/Dockerfile.operator#L25

Reply:
ah right. Let me double check but I think you are correct

Reply:
I think we can resolve this conversation?